opentelekomcloud/common/importers.go

```diff
@@ -1,8 +1,12 @@
 package common
 
-import "github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+import (
+	"context"
 
-func ImportAsManaged(d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ImportAsManaged(ctx context.Context, d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
```

Comment:
Should be `func ImportAsManaged(_ context.Context, d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {`